src/connectors/indie88.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+// Indie88 has the track and artist flipped on their main web player. The values are intentionally flipped in this Connector
+
+Connector.playerSelector = 'div.cobrp-player-main, div.mediaplayer-mainbody';
+
+Connector.artistSelector = 'div.cobrp-player-main div.cobrp-player-artist, div.mediaplayer-mainbody .mediaplayer-snp p#track-info-title span';
```

Comment:
You could use array of selectors here for more readability and safer behavior:
```js
Connector.artistSelector = [
	'selector-1',
	'selector-2',
];
```

Also, these selectors could be simplified, e.g.: 
`div.cobrp-player-main div.cobrp-player-artist` -> `.cobrp-player-artist`